site/docs/pages/index.mdx

```diff
@@ -100,7 +100,7 @@ bun add @coinbase/onchainkit
 :::
 
     </div>
-      <a href="/getting-started" title="View OnchainKit docs" className="font-sans border rounded-xl border-solid border-indigo-400 text-indigo-400 px-4 py-3 font-bold mt-4">
+      <a href="/getting-started" title="View OnchainKit docs" className="font-sans border rounded-xl border-solid border-indigo-600 text-indigo-600 hover:border-indigo-700 dark:border-indigo-400 dark:text-indigo-400 dark:hover:border-indigo-300 px-4 py-3 font-bold mt-4">
```

Comment:
It's really bothering me how long these single lines are getting. I will clean up in a future PR - I don't want to mix up all of that clean up with actual changes / updates.